DESCRIPTION

```diff
@@ -6,6 +6,46 @@ Author: Displayr <opensource@displayr.com>
 Maintainer: Displayr <opensource@displayr.com>
 Description: Utility functions to load all flip packages.
 License: GPL-3
+Suggests:
+    flipAnalysisOfVariance,
```

Comment:
Where are `flipU`, `flipTime`, `flipTransformations`, `verbs`, and `flipTables`? Also, `attach_list` in the R code mentions `lubridate`.

Reply:
> Could list the packages from load_list in Suggests?

These generated

```
 flipFlip:::load_list
 [1] "flipStatistics"         "flipImputation"         "flipData"               "flipChartBasics"       
 [5] "flipAPI"                "flipPictographs"        "flipStandardCharts"     "flipPlots"             
 [9] "flipDashboard"          "flipDimensionReduction" "flipRegression"         "flipStartup"           
[13] "flipTrees"              "flipAnalysisOfVariance" "flipQualtrics"          "flipMultivariates"     
[17] "flipCluster"            "flipTURF"  
```
Those packages have been added to `Suggests` as requested.

It passes the check which is what was desired I thought. Does the `attach_list` need to be account for as well? If so, why?

Reply:
Here are the list of packages that generate the `R CMD check` warning. It includes the ones I've mentioned. 
![image](https://user-images.githubusercontent.com/5276419/234530378-e197e129-9ebd-4223-a969-84b11dbed954.png)

Additionally, by having every package listed in Suggests, `remotes::install_deps` will install every package that `flipFlip` expects to be installed/available.

Reply:
https://app.circleci.com/pipelines/github/Displayr/flipFlip/13/workflows/81096f48-694b-4914-ae31-7dbb179a8cf1/jobs/14 doesn't generate any warnings. So something isn't right with CircleCI

Reply:
What screenshot is that from, the line numbers don't match CircleCI.

Reply:
All of the additional dep requests are installed by CircleCI since they are required for packages in the `load_list` https://circleci.com/api/v1.1/project/github/Displayr/flipFlip/14/output/105/0?file=true&allocation-id=64423620ad230f5f4ef1defe-0-build%2FQ8DEFH45

Reply:
That is from travis: https://app.travis-ci.com/github/Displayr/flipFlip/builds/258087273
The CCI build wouldn't need those packages explicitly listed in Suggests because they would be installed as dependencies by `install_deps()` while installing the packages you've already listed.

Reply:
There is no problem from what I can see. What am I missing?

Reply:
@mwmclean

Reply:
Your approach could conceivably break if one of the missing packages is dropped as a dependency by the packages that are listed. For R programmers that like to read the DESCRIPTION file to get an overview of a package, we provide an incomplete, inaccurate list of the packages in our flip-verse.